src/Files.App/Strings/en-US/Resources.resw

```diff
@@ -1611,11 +1611,11 @@
   <data name="SecurityListDirectoryLabel.Text" xml:space="preserve">
     <value>List directory contents</value>
   </data>
-  <data name="SecurityModifyLabel.Text" xml:space="preserve">
+  <data name="Modify" xml:space="preserve">
```

Comment:
Changing this string will reset the translations, can you revert this change or update it in the other resource files?

Reply:
What was the error?

Reply:
PRI175: 0x80073b39 - Processing Resources
PRI278: 0x80073b39 - 'Resources/SecurityOwnerLabel' or one if its parents is defined as both resource and scope, which is not allowed.

https://github.com/files-community/Files/runs/12729877592

Reply:
Debugging this could take a while, I'll approve this change so we can move on.